Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,11 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * @return string
+     */
+    abstract protected function getContext();
```

Comment:
> @Soullivaneuh : instead of commenting, the method could be made not abstract at first, then abstract on the master branch.

Commenting the signature is to not forget to add it on the next major.